### ISSUE TITLE
Web Console in 1.15.10 getting spammed by VRMsg_ logs.

### ISSUE THREAD

**Issue by risa2000** [ValveSoftware/openvr#1478]
Would it be possible to disable some components/events logging in Web Console?

The latest version 1.15.10 is literally spamming the Web Console with the messages of type:
```
Processing Message of type VRMsg_ComputeDistortion (12 bytes) from steamtours (17968)
```
making the SteamVR startup vanish.

An example snippet out of the web console log:
[web_console_spam.txt](https://github.com/ValveSoftware/openvr/files/5578682/web_console_spam.txt)

**Comment by okawo80085** [comment]
oh yeah  i noticed that too, it doesn't happen every boot tho, maybe they forgot to toggle some debug option back?

**Comment by JoeLudwig** [comment]
This should be fixed in 1.15.13
